### DISCUSSION TITLE
Is fake-bpy-module-4.1 nearing availability?

### DISCUSSION THREAD

**Discussion by ZeroGravitasIndeed** [nutti/fake-bpy-module#199] [Q&A] [answered]
As previous releases seemed to closely follow the matching Blender versions' ones, I was wondering whether fake-bpy-module-4.1 is nearing release or 4.0 is supposed to cover 4.1's features already.

(I was about to give your recipe to autogenerate it a try, which would be my very first. In any case, I wish to state my gratefulness for these works of yours 🙂)

**Comment by nutti** [comment, marked as answer]
@ZeroGravitasIndeed 

We are now refactoring the whole code to increase the readability and maintenance easily.
You can use latest version by `pip install fake-bpy-module` for now instead.
https://pypi.org/project/fake-bpy-module/
